test/testHelper.js

```diff
@@ -0,0 +1,36 @@
+module.exports = {
+  testRole: {
+    title: 'admin'
+  },
+
+  testRole2: {
+    title: 'regular'
+  },
+
+  testUser: {
+    userName: 'oredavids',
+    firstName: 'ore',
+    lastName: 'davids',
+    email: 'oredavids@gmail.com',
+    password: '123456'
+  },
+
+  testUser2: {
+    userName: 'femi',
+    firstName: 'femi',
+    lastName: 'dotexe',
+    email: 'femidotexe@gmail.com',
```

Comment:
Why not just use [`faker`](https://www.npmjs.com/package/Faker), I must not receive spam mails ooooo